gui/boosterView.py

```diff
@@ -27,7 +27,8 @@
 
 
 class BoosterViewDrop(wx.PyDropTarget):
-    def __init__(self, dropFn):
+    def __init__(self, dropFn, *args, **kwargs):
+        super(BoosterViewDrop, self).__init__(*args, **kwargs)
```

Comment:
>All we do (usually) is define the parent window, and then call the parent with the required parameters (along with our implementation of optional parameters depending on what we need for this Window). 

And in this case, I would agree with the argument made here:
http://stackoverflow.com/questions/2294200/issue-with-python-class-hierarchy/2295784#2295784

If we're not doing what the parent is doing, then why are we inheriting the parent?  If we need to wrap it to do something else, we can do that then call the parent.

If we are doing what the parent is doing....then why do we have different parameters?

>Take all of our wxWindow definitions. We inherit wx.Panel, wx.Frame, etc, but we don't define the same things the parent defines "just in case we need them in the future". 

Especially in the case where we're extending (and not replacing) a default class/function, we can get ourselves into trouble.  There's a rather good explanation here:
http://stackoverflow.com/questions/2398666/python-can-subclasses-overload-inherited-methods

Ultimately,  I think our future selves will thank us for this.  It all boils down to two main arguments.

Because inspection tells us to.  Now, inspection isn't perfect (it has a hard time following `wx` for example) but it's pretty damn good.  We currently have 49 warnings where the signatures don't match.  If we don't clean those up, then we have a few  choices.
1) Exception out the check entirely.  This I don't like, because it's definitely a good smell test that something might not be right.
2) Exception out individual classes/methods.  This is better than option 1,  but then the temptation becomes "just add the exception  to everything".
3) Leave it as is.  Then we just start ignoring the warnings, and might as well have done option 1.

The second argument is a little simpler.  Because  our future selves (and future devs) won't go hunting for monkeypatches when they are looking to implement something.  When I'm looking at docs, I go back to the Python/wx/whatever docs.  If something then doesn't behave the way I expect it to, I will first  assume it's my code or lack of understanding, not a monkeypatch that causes it to not be handled it the way the docs say it will.  That means I'm wasting time/effort and is _incredibly_ frustrating.

Yes, this means that we'll have definitions with `*args` and `**kwargs`, but there's no way around that unless you  want to entirely just throw out that check or handle the reference back to it differently (explicitly call it instead of inheriting it).

And as far as "littering" goes, we're really only talking about about a handful of lines here,  and primarily will just be in the `__init__`.  This isn't a change of 2000 lines, probably more like 40-50.

There  _are_  a few  functions that we might need wider changes. `populatePanel` is pretty inconsistent, for example, and for those we should probably tackle them outside this PR, since that might entail wider code changes.  But for just explicitly inheriting base classes, that's not a big deal. 

>I believe we should be good to go with this PR. :)

I'm going to go through another pass of the warnings and make sure there's nothing  else we missed.  Inspection caught a missing import in `fittingView`, so I don't think we're _quite_ there yet (but damn close).